src/core/src/op/einsum.cpp

```diff
@@ -77,8 +70,28 @@ bool is_label_elsewhere(const std::vector<std::string>& input_subscripts,
     return false;
 }
 
+/// \brief Remove all whitespaces from given string
+///
+/// \param[in,out] s  String to process.
+///
+void remove_whitespaces(std::string& s) {
+    s.erase(std::remove_if(s.begin(),
+                           s.end(),
+                           [](unsigned char c) {
+                               return std::isspace(c);
+                           }),
+            s.end());
+}
+
 }  // namespace
 
+BWDCMP_RTTI_DEFINITION(op::v7::Einsum);
```

Comment:
`BWDCMP_RTTI_DEFINITION` macro has been removed from all operations, please update the branch and remove it as well to avoid conflicts.
- https://github.com/openvinotoolkit/openvino/pull/14292